File: sdk/src/main/java/com/cxense/cxensesdk/model/WidgetResponse.java
package com.cxense.cxensesdk.model;

import androidx.annotation.Nullable;

import java.util.List;

/**
 * Response for widget from server.
 *
 * @author Dmitriy Konopelkin (dmitry.konopelkin@cxense.com) on (2017-06-05).
 */

public final class WidgetResponse {
    /**
     * widget items
     */
    @Nullable
    public List<WidgetItem> items;
    /**
     * widget template
     */
    @Nullable
    public String template;
    /**
     * widget style
     */
    @Nullable
    public String style;
}

File: sdk/src/main/java/com/cxense/cxensesdk/model/WidgetResponse.kt
package com.cxense.cxensesdk.model

import com.squareup.moshi.JsonClass

/**
 * Response for widget from server.
 */
@JsonClass(generateAdapter = true)
class WidgetResponse(
    val items: List<WidgetItem> = emptyList(),
    val template: String? = null,
    val style: String? = null
)
